CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Next
+* Added support for node-meta param to specify node level tags when queying for service health. 
+  eg Diplomat::Health.service('apache', passing: true, node_meta: "rack-2")
```

Comment:
Doc is wrong, format is:
```
node_meta: "rack:rack-2"
```
as described here: https://www.consul.io/api-docs/health#node-meta